src/material/datepicker/calendar-body.html

```diff
@@ -60,6 +60,6 @@
         [class.mat-calendar-body-today]="todayValue === item.compareValue">
         {{item.displayValue}}
       </div>
-      <div class="mat-calendar-body-cell-preview"></div>
+      <div class="mat-calendar-body-cell-preview" aria-hidden="true"></div>
```

Comment:
Does this behavior alternatively go away if we remove the `content` css property from `.mat-calendar-body-cell-preview`? 

@crisbeto do you remember why this rule has the `content` property?

Reply:
I'm guessing it's because I wanted to reuse the same styles with some `::before` and `::after` selectors and I didn't want to split it out only due to the `content`: https://github.com/angular/components/blob/master/src/material/datepicker/calendar-body.scss#L54.

This is a little bit surprising to me, I assumed that `content` only works on pseudo elements.

Reply:
Interesting note from MDN on the `content` property:

> CSS-generated content is not included in the DOM. Because of this, it will not be represented in the accessibility tree and certain assistive technology/browser combinations will not announce it. If the content conveys information that is critical to understanding the page's purpose, it is better to include it in the main document.

https://developer.mozilla.org/en-US/docs/Web/CSS/content#accessibility_concerns

Reply:
Hmm, I tried removing the CSS content in #24095, but that did not fix the duplicate nav stop with VoiceOver

Reply:
Weird, I wouldn't have expected an empty `div` to be picked up by a screen reader. I don't mind going with the `aria-hidden` here, but it would be good to understand why it happens.